Assets/Scripts/SimplePhysics.cs

```diff
@@ -255,5 +257,21 @@ private bool RaycastY(ref float delta, float dir, int width)
             }
             return false;
         }
+
+        private bool TwoWayPlatform(GameObject other)

```

Comment:
On what conditions does this return true. It seems like it's saying that you return true if this is a two way platform you should fall through (i.e. its a two way and you've pressed down or are below it) and return false if you should collide with it (i.e. you are falling on the two way without pressing down or its not a two way). Am I correct?